extension/jmxmetricextension/README.md

```diff
@@ -40,17 +48,25 @@ Corresponds to the `otel.jmx.service.url` property.
 
 _Required._
 
+### target_system
+
+The built-in target system metric gatherer script to run.
+
+Corresponds to the `otel.jmx.target.system` property.
+
+One of `groovy_script` or `target_system` is _required_.  Both cannot be specified at the same time.
+
 ### groovy_script
 
 The path of the Groovy script the Metric Gatherer should run.
 
 Corresponds to the `otel.jmx.groovy.script` property.
 
-_Required._
+One of `groovy_script` or `target_system` is _required_.  Both cannot be specified at the same time.
 
-### interval (default: 10s)
+### interval (default: `10s`)
 
-The interval time for the Groovy script to be run and metrics exported.  Will be converted to milliseconds.
+The interval time for the Groovy script to be run and metrics exported.
```

Comment:
Might be answered later on, but is it starting the JVM every 10s just to gather the JMX metrics? Or is the JVM started once? It might be good to mention it here.

Reply:
This in turns sets the property that determines how frequently the custom or target groovy script is run.  The JVM process persists through the lifetime of the extension, as well as its JMX connection and any otel meter.

Specified description a bit.